dashboard/app/models/user.rb

```diff
@@ -2658,6 +2658,28 @@ def code_review_groups
     end
   end
 
+  # Is this user compliant with our Child Account Policy(cap)?
+  # For students under-13, in Colorado, with a personal email login: we require
+  # parent permission before the student can start using their account.
+  def cap_compliant?
+    age >= 13 || us_state != 'CO' || !personal_account? ||
+      child_account_compliance_state == ChildAccountCompliance::PERMISSION_GRANTED
```

Comment:
Just a small suggestion on how to make it more readable
```suggestion
    return true if age >= 13
    return true unless us_state == 'CO'
    return true unless personal_account?

    child_account_compliance_state == ChildAccountCompliance::PERMISSION_GRANTED
```

Reply:
Good idea. I changes it a little by making the configuration a hash rather than a single integer just incase we want to add other restrictions as well